src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/mgmt-utils.c

```diff
@@ -613,11 +610,13 @@ static void xclmgmt_reset_pci(struct xclmgmt_dev *lro)
 
 int xclmgmt_update_userpf_blob(struct xclmgmt_dev *lro)
 {
-	int len, userpf_idx;
-	int ret;
-	struct FeatureRomHeader	rom_header;
-	int offset;
-	const int *version;
+	int len = 0;
+	int userpf_idx = 0;
+	int ret = 0;
+	struct FeatureRomHeader rom_header;
```

Comment:
Structure not initialized
```suggestion
	struct FeatureRomHeader rom_header = {};
```